camp_2/README.md

```diff
@@ -304,16 +304,28 @@ protostar -p devnet invoke \
     --max-fee auto
 ``` 
 
-This was the long way to deploy the voting contract, similar to what we did with Argent's UI and the `starknet` CLI. However, Protostar is a fantastic tool; we can use the `deploy` command to deploy our contract without invoking the UDC; Protostar does it in the background.
+This was the long way to deploy the voting contract, similar to what we did with Argent's UI and the `starknet` CLI. However, Protostar is a fantastic tool; we can use the `deploy` command to deploy our contract without invoking the UDC; Protostar does it in the background. This time let's use the accounts that the devnet previously created for us.
 
 ```Bash
-protostar -p devnet deploy 0x067b2b9d2a45c662edad93edf0c10bc3715f315cfa3a6e530f1cf1c5377989ed \
+protostar -p devnet deploy 0x02bebbf6d9e8cf11a1d0e04226e68f8d73ddfb025fb058c769fc6b48b27e57cf \
     --account-address 0x7e00d496e324876bbc8531f2d9a82bf154d1a04a50218ee74cdd372f75a551a \
     --max-fee auto \
-    --inputs 111 2 222 333
+    --inputs \
```

Comment:
Are we a 100% sure that we can use  the`\` separator between a flag and its value? I think it's not possible to use it in that position but I might be wrong. Might be worthwhile copy pasting the code as is on a terminal to see if it's accepted

Reply:
yes! It works, I tried it before writing it.